packages/web/src/index.css

```diff
@@ -37,6 +38,7 @@
     --sb-red-20: 0 100% 86%;
     --sb-red-30: 0 100% 76%;
     --sb-red-40: 0 100% 66%;
+    --sb-red-40-subtle: 0 100% 66% 0.1;
```

Comment:
Why are the existing colors not sufficient here?

Reply:
needed alpha channel variant, got rid of these and moved to `hsl(var(--sb-*-*) / 0.1)`